Anima/diffusion/mcm/animaNODDICompartment.h

```diff
@@ -111,7 +111,7 @@ class ANIMAMCM_EXPORT NODDICompartment : public BaseCompartment
     Vector3DType m_CurrentGradient;
     
     // Internal work variables for faster processing
-    std::vector<double> m_WatsonSHCoefficients, m_WatsonSHCoefficientDerivatives;
+    std::vector <double> m_WatsonSHCoefficients, m_WatsonSHCoefficientDerivatives;
```

Comment:
I understand this. It was troubling to see no whitespace here.

Reply:
:)